snap/snapcraft.yaml

```diff
@@ -193,13 +195,3 @@ parts:
       - licenses/LICENSE-snap
     organize:
       LICENSE: licenses/LICENSE-snap
-  percona-mongodb-exporter-license:
-    plugin: nil
-    build-packages:
-      - wget
-    override-build: |
-      PERCONA_GH=https://raw.githubusercontent.com/percona
-      craftctl default
-      mkdir -p $CRAFT_PART_INSTALL/licenses
-      wget ${PERCONA_GH}/mongodb_exporter/v0.40.0/LICENSE \
-        -O $CRAFT_PART_INSTALL/licenses/LICENSE-mongodb-exporter
```

Comment:
i believe we may still need to include the license for the exporters in the snap. see [how it is done](https://github.com/canonical/charmed-mongodb-snap/pull/54/files#diff-56759910381a014fecfd7556dd72ddd68c747d922a5b7df2044b9ce7c552f5f5L177-L180) for other packages in this snap. this organize section can be added to the `percona-mongodb-exporter` step